Code/Tools/SceneAPI/SceneUI/Handlers/ProcessingHandlers/AsyncOperationProcessingHandler.h

```diff
@@ -34,14 +35,14 @@ namespace AZ
                 ~AsyncOperationProcessingHandler() override = default;
                 void BeginProcessing() override;
 
-            private:
+            private Q_SLOTS:
                 void OnBackgroundOperationComplete();
 
             private:
                 AZ_PUSH_DISABLE_DLL_EXPORT_MEMBER_WARNING
                 AZStd::function<void()> m_operationToRun;
                 AZStd::function<void()> m_onComplete;
-                AZStd::unique_ptr<AZStd::thread> m_thread;
+                QThread* m_thread = nullptr;
                 AZ_POP_DISABLE_DLL_EXPORT_MEMBER_WARNING
```

Comment:
Author's note:  The crash this caused was in a QThreadStorage object being used on a non-qt thread, for the QtFreeType font library.   By making this already-qt-object use only qt-thread primitives, the proper thread storage setup and teardown prevents a crash.